src/sdk/filters/getGroupBy.js

```diff
@@ -0,0 +1,29 @@
+const byChartDefaultGroupBy = {
+  "postgres.db_transactions_ratio": "database",
```

Comment:
We sum committed and rollback transactions ratio here when aggregating per db. Not sure it makes sense. cc @shyamvalsan

Reply:
Agree @ilyam8 we should not group this by database - does not make sense.